websites/P/Pass Culture/presence.ts

```diff
@@ -0,0 +1,140 @@
+const presence = new Presence({
+	clientId: "1165384499987742800",
+});
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "https://i.imgur.com/ES9Hy84.png",
+		},
+		path = document.location.pathname.split("/"),
+		showCounts = await presence.getSetting<boolean>("show_counts");
+
+	switch (path[1]) {
+		case "questionnaire-pratiques-initiales":
+			presenceData.details = "En train de faire le questionnaire";
+			break;
+		case "accueil":
+			presenceData.details = "Regarde la page d'accueil";
+			break;
+		case "accueil-thematique": {
+			presenceData.details = "Regarde la page d'accueil";
+
+			const offersType = (<HTMLHeadingElement>(
+				document.querySelector(".css-175oi2r.r-43z982.r-1udh08x")
+			))?.textContent;
```

Comment:
Use the generics rather than casting:
```ts
document.querySelector<HTMLHeadingElement>(...);
```

Also, try to avoid selectors that use computer-generated ids if possible.